packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -75,6 +78,14 @@ const exchageTransitionConfigOpts: ReqOpts = {routeId: "exchangeTransitionConfig
 export class ExecutionEngineHttp implements IExecutionEngine {
   readonly payloadIdCache = new PayloadIdCache();
   private readonly rpc: IJsonRpcHttpClient;
+  private readonly jobQueue: JobItemQueue<[EngineRequest], EngineResponse>;
```

Comment:
Since it's not generic to multiple jobs I would call it `rpcFetchQueue` or something like that. Also add a comment explaining why this is necessary and reason which methods need it and which do not